core/modules/css/CssAccess.cc

```diff
@@ -579,8 +522,20 @@ CssAccess::getTableParams(std::string const& dbName, std::string const& tableNam
         if (iter != paramMap.end()) {
             params.partitioning.overlap = std::stod(iter->second);
         }
+        iter = paramMap.find("partitioning/lockInMem");
+        if (iter != paramMap.end()) {
+            params.partitioning.lockInMem = std::stoi(paramMap["partitioning/lockInMem"]);
+        } else {
+            params.partitioning.lockInMem = false;
```

Comment:
TYhe default value should be set by default constructor, you don't really need it here.